### ISSUE TITLE
Dsc_xpackage parameter issues

### ISSUE THREAD

**Issue by marvin5881** [dsccommunity/xPSDesiredStateConfiguration#762]
### Problem description

Powershell is having an issue with a certain parameter and is requesting to use one that is not a part of the Dsc_xpackage resource according to the documentation. It seems to want me to use the Parameter "Path" but the closest thing I can see in the documentation is "dsc_path"

### Verbose logs

```text
no parameter named 'path' on Dsc_xpackage[sophos_install_agent]

dsc_xpackage: PowerShell DSC resource DSC_xPackageResource failed to execute Get-TargetResource functionality with error message: The specified Path 'D:\CMMain8' does not appear to specify an EXE or MSI file and as such is not supported.
Parameter name: Path
```

### DSC configuration

```powershell
dsc_xpackage { 'sophos_install_agent':
        dsc_ensure    => present,
        dsc_name      => 'Sophos Endpoint Agent',
        #dsc_productid => '',
        dsc_path      => $test,
        dsc_arguments => '--quiet',
        require       => Reboot['reboot_when_refreshed_exclude_wau_rename'],
        notify        => Reboot['sophos_reboot_when_refreshed'],
      }
```

### Suggested solution

Attempt to replicate issue in a different envirionment.

### Operating system the target node is running

```text
Windows Server 2019
```

### PowerShell version and build the target node is running

```text
PowerShell 7
```

### xPSDesiredStateConfiguration version

```text
9.1.0-0-4
```

**Comment by johlju** [comment]
I think this might be an issue related to Puppet? 

This is tested using an integration test, and very much uses a parameter Path
https://github.com/dsccommunity/xPSDesiredStateConfiguration/blob/7703bb11324d26c97d80684bdc4fe46cb982b0ca/tests/Integration/DSC_xPackageResource.Integration.Tests.ps1#L93-L99

But it might be related that Puppet converts `dsc_path` to the resource parameter ´Path`?

**Comment by marvin5881** [comment]
Yes, I am using Puppet to automate DSC via the module provided by this community. `Path` isn't something that I can configure directly with Puppet but it would make sense that it's converted from `dsc_path`. I'm not quite sure if it matters that `dsc_path` is labeled as a Property rather than a parameter.

https://forge.puppet.com/modules/dsc/xpsdesiredstateconfiguration/reference#dsc_xpackage

Is there a workaround for this?

**Comment by johlju** [comment]
I think you do best by reaching out to a Puppet forum as this cannot be fixed in this repo.  You could also ask in the DSC-channel in Slack/Discord to see if there are any Puppet users that can help: https://dsccommunity.org/community/contact/
